CONTRIBUTING.md

```diff
@@ -68,10 +68,18 @@ poetry run flake8 ./xrpl
 
 ### Running Tests
 
+#### Individual Tests
+
+```bash
+# Works for single or multiple unit/integration tests
+# Ex: poetry run poe test tests/unit/clients/test_json_rpc_client.py tests/integration/transactions/test_account_delete.py
```

Comment:
super nit: the client tests might be moving to integration tests later, I'd pick a different test to use as your example